x-pack/plugins/uptime/public/components/functional/ping_list.tsx

```diff
@@ -60,14 +60,14 @@ export const PingList = ({
               })}
         </EuiHealth>
       ),
-      sortable: true,
+      sortable: false,
     },
     {
       field: 'timestamp',
       name: i18n.translate('xpack.uptime.pingList.timestampColumnLabel', {
         defaultMessage: 'Timestamp',
       }),
-      sortable: true,
+      sortable: false,
```

Comment:
Question - do you think we should explicitly set this to `false` for all of them? Or just remove the `sortable` prop for the ones that currently have it? Right now the ones that bear a definition of this prop are somewhat arbitrary, and as it defaults to `false` from an organizational standpoint we should probably treat all the columns the same way. WDYT?

Reply:
I like making it explicit for now, since it indicates it's a conscious choice.

Reply:
Do you think we should add the `false` declaration to the remaining column definitions?

Reply:
Oh, sorry, yes. I'll make that consistent.

Reply:
++ - as I noted in [my comment](https://github.com/elastic/kibana/pull/30430#issuecomment-461864501) the functionality LGTM, so after you add that update we'll be good to go.